### PR TITLE
Create ui-c8y-1020-38-0-alert-text-improvement-7222-7345.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61028](https://cumulocity.atlassian.net/browse/MTM-61028)
Original PR: [7345](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7345)

[MTM-61028]: https://cumulocity.atlassian.net/browse/MTM-61028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-38-0-alert-text-improvement-7222-7345.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: alert text improvement (#7222) (#7345)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-38-0-alert-text-improvement-7222-7345.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved alert text for better clarity
```

File: content/change-logs/platform-services/ui-c8y-1020-38-0-alert-text-improvement-7222-7345.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61028
+version: 1020.38.0
+---
+alert text improvement (#7222) (#7345)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-38-0-alert-text-improvement-7222-7345.md:17]: The Claude AI suggested to use the following body:
```suggestion
To provide users with more meaningful information about alerts, the alert text has been improved. Previously, the alert text was often unclear or confusing, making it difficult for users to understand the nature of the issue. With this change, the alert text now includes additional context and details to help users quickly identify and resolve the problem. This improvement should lead to a better user experience and reduced confusion when encountering alerts.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@kowa-c8y Please review and approve this entry.

**Comment by mmed-c8y** [comment]
Release notes are not needed - the fix changes the content of the alert shown to tenants with shared trusted certificate enabled.

**Pull request closed without merging**